tests/data/conf_covariance_ssc_fsky.yaml

```diff
@@ -0,0 +1,48 @@
+tjpcov:
+  # sacc input file 
+  sacc_file: ./tests/benchmarks/32_DES_tjpcov_bm/cls_cov.fits
+
+  # 'set' from parameters OR pass CCL cosmology object OR yaml file
+  cosmo: 'set'
+
+  # Setting mask OR fsky approximation
+  mask_file:
```

Comment:
I forgot: to test the fsky approx properly, when no masks are passed. Remove these lines. I think the code worked because you had these, but, without them, it would fail (because of the `get_masks` thing